network/ocsp.go

```diff
@@ -208,6 +212,8 @@ func (v DefaultOCSPVerifier) Verify(chain []*x509.Certificate) (int, error) {
 	confirmsByConfigOCSP := 0
 	confirmsByCertOCSP := 0
 
+	// TODO avoid querying same OCSP more than once
```

Comment:
yeah,  nice idea. probably to use some `map[string]struct{}{}` as set of unique values.